app/server/lib/ConfigBackendAPI.ts

```diff
@@ -0,0 +1,67 @@
+import * as express from 'express';
+import {expressWrap} from 'app/server/lib/expressWrap';
+
+import {IGristCoreConfig} from 'app/server/lib/configCore';
+import {getGlobalConfig} from 'app/server/lib/globalConfig';
+
+import log from "app/server/lib/log";
+
+export class ConfigBackendAPI {
+  private static _doRestart = false;
+  private _config: IGristCoreConfig;
+
+  constructor() {
+      this._config = getGlobalConfig();
+  }
+
+  public addEndpoints(app: express.Express, requireMiddleware: express.RequestHandler) {
```

Comment:
Can you add comments? What is this requierMIddleware?

Reply:
It should always be the `requireInstallAdmin` middleware. I've renamed the variable to reflect this.